tests/test_port_dpb_system.py

```diff
@@ -167,6 +168,49 @@ def test_port_breakout_one(self, dvs):
         self.verify_only_ports_exist(dvs, ["Ethernet0"])
         print "**** 2x25G(2)+1x50G(2) --> 1x100G passed ****"
 
+    def test_port_breakout_with_vlan(self, dvs):
+        dvs.setup_db()
+        portName = "Ethernet0"
+        vlanID = "100"
+        breakoutMode1 = "1x100G[40G]"
```

Comment:
we can use before(bfrBreakOutMode) and after(afrBreakOutMode) too . Also, what about having one more test, where Say Ethernet4 is also part of same VLAN and an additional check is that if Ethernet4 not removed from VLAN.

Reply:
Agreed, we can make all different combinations and have lot more test cases. Some of them especially the one that you are describing is covered in test_port_dpb_vlan.py. Here all that I am covering is very basic VLAN dependency test case using CLI command. More will be covered while exercising other options of CLI comamnd.

Reply:
BTW, I did think about before and after breakout modes. But they do NOT make sense as I am also doing vice-versa.